compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/BIREmitter.java

```diff
@@ -283,10 +303,25 @@ public void visit(BIROperand birOp) {
         sb.append(birOp.variableDcl.name);
     }
 
+    // Positions
 
     private void writePosition(DiagnosticPos pos) {
-        sb.append("\t\t// pos:[").append(pos.sLine).append(":").append(pos.sCol).append("-");
-        sb.append(pos.eLine).append(":").append(pos.eCol).append("]");
+        if (pos != null) {
+            appendPos(sb, pos.sLine);
+            sb.append("-");
+            appendPos(sb, pos.eLine);
+            sb.append(":");
+            appendPos(sb, pos.sCol);
+            sb.append("-");
+            appendPos(sb, pos.eCol);
+        }
+    }
+
+    private void appendPos(StringBuilder sb, int line) {
+        if (line < 10) {
+            sb.append("0");
```

Comment:
What is the logic behind this?

Reply:
This is to print the position data with two digits with bir dump option mainly for readability purpose. 

ex. 
04-04:21-22	%2 = CONST_LOAD <int> 2;
04-04:24-25	%3 = CONST_LOAD <int> 3;